libs/atqr/domain/src/lib/EmailService/templates/templateMessage.ts

```diff
@@ -0,0 +1,75 @@
+import { ChallengeAndInviteeEnvelope, ChallengeEnvelope } from '../dtos';
+import { ITemplateMessage } from './interfaces/templateMessage.interface';
+
+export enum NotificationChannel {
+  Email = 'Email',
+  SMS = 'SMS',
+  Push = 'Push',
+}
+// create an enum for each type of content
+export enum ChallengeEnum {
```

Comment:
I don't know if this implementation is the right one or the most efficient one to relate the emails to the content they should contain. @yfernandes